gravitee-apim-portal-webui-next/src/app/api/api.component.html

```diff
@@ -0,0 +1,19 @@
+<!--
+
+    Copyright (C) 2024 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<app-breadcrumb-navigation></app-breadcrumb-navigation>
```

Comment:
Since the html is super simple, this could have also just been directly in the `.component.ts` file. But this is fine as is 👍

Reply:
true